### ISSUE TITLE
tour: automatically terminate execution when error occurs

### ISSUE THREAD

**Issue by CaymanTurtleBeach** [golang/tour#1362]
Context: https://go.dev/tour/basics/3

The text (instructions) do NOT mention the "Kill" button. It just magically appears after initially running the code that contains the error (running in Brave browser on an x64 Win 10 computer).

But when reading the instructions linearly (how else??):

 ... preliminaries
```
Run the code. Notice the error message.
To fix the error, rename math.pi to math.Pi and try it again.
```
User makes change, but how to try again?? The visible actions are:
`Reset Format Kill`
Reset (maddeningly to someone who just fixed the error) of course restores the error to the code,

What is **Kill**? A neophyte who starts the tour may or may not be a *nix weenie and thus may not intuit that a process must be killed, and a real neophyte (basically no coding) who is gamely trying to learn will be stymied until figuring out what to do. But why impose trial and error in a tutorial??

My opinion would be to mirror what happens on a local command line - have the program exit after the error is detected/displayed, like with (`go run .`).
In other words, display the error message and "exit" in the Tour (i.e., leave the Run button labeled **Run**. The user sees error, fixes it, and clicks run again.
If exiting execution is not an option (why??), then add verbiage about needing to click Kill before the program can be run again - though I stress that I think just exiting the execution and leaving button labels alone is preferable.
